packages/app/screens/onboarding/OnboardingStack.js

```diff
@@ -0,0 +1,197 @@
+import React, { useState, useRef } from 'react';
+import { View, Text, TouchableOpacity, TextInput, StyleSheet, ScrollView, Dimensions, Animated } from 'react-native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { useUserContext } from '@app/utils/UserContext';
+
+const Stack = createNativeStackNavigator();
+
+function OnboardingScreen({ navigation }) {
+  const [school, setSchool] = useState('');
+  const [birthday, setBirthday] = useState('');
+  const [interests, setInterests] = useState('');
+  const [error1, setError1 ] = useState(false);
+  const [error2, setError2 ] = useState(false);
+  const [error3, setError3 ] = useState(false);
+  const [error4, setError4 ] = useState(false);
+
+  const { onboardingCompleted } = useUserContext();
+
+  const handleSubmit = async () => {
+    await AsyncStorage.setItem('isFirstTime', JSON.stringify(false));
+    onboardingCompleted();
+
+    if (!school.trim())
+    {
+      setError1(true);
+    }
+    if (!birthday.trim())
+    {
+      setError2(true);
+    }
+    if (!interests.trim())
+    {
+      setError3(true);
+    }
+    if (!school.trim() || !birthday.trim() || !interests.trim())
+    {
+      setError4(true);
+    }
+    else
+    {
+      navigation.navigate('TabNavigation');
+    }
+  };
+
+  const handleSkip = async () => {
+    navigation.navigate('TabNavigation');
+  };
+
+   return (
+    <View>
+      <ScrollView 
+      pagingEnabled={true} 
+      horizontal={true}
+      showsHorizontalScrollIndicator={false}>
+        <View style={styles.slide}>
+          <Text>Where do you go to school?</Text>
+          <TextInput
+          style={styles.input}
+          placeholder= "Your School"
+          onChangeText={setSchool}
+          value={school}
+          />
+          {error2 ? (
+            <Text style={styles.error}>*Your School is Required</Text>
+            ) : null
+          }
+        </View>
+
+        <View style={styles.slide}>
+          <Text>When's your birthday?</Text>
+          <TextInput
+          style={styles.input}
+          placeholder= "MM/DD/YYYY"
+          onChangeText={setBirthday}
+          value={birthday}
+          />
+          {error2 ? (
+            <Text style={styles.error}>*Your Birthday is Required</Text>
+            ) : null
+          }
+        </View>
```

Comment:
There should be input validation checking if the correct values are typed in for Birthday (maybe School too?). A date picker would also be a good choice for the Birthday.